haskell/Haskell.g4

```diff
@@ -0,0 +1,2366 @@
+grammar Haskell;
+
+@lexer::members {
```

Comment:
We can put them to superClass, just what php grammar did.

Reply:
Not only [PHP](https://github.com/antlr/grammars-v4/tree/master/php), but also [JavaScript](https://github.com/antlr/grammars-v4/tree/master/javascript/javascript), [Golang](https://github.com/antlr/grammars-v4/tree/master/golang).